presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    extra_hosts:
+       - "${REPLACE_LDAP_SERVER_ADDRESS}:${REPLACE_LDAP_SERVER_IP}"
```

Comment:
We misunderstood each other :) I meant the variables to be named without the 'replace' prefix + there should be an `environment` section, as below:

```
environment:
  LDAP_SERVER_ADDRESS: REPLACE_LDAP_SERVER_ADDRESS
  LDAP_SERVER_IP: REPLACE_LDAP_SERVER_IP
```

Or, if we want to avoid the repetition, the value of both vars can be 'REPLACE_ME'. So that user get's a 'no such host error: REPLACE_ME' if they don't specify the vars.